hoomd/hpmc/ExternalFieldHarmonic.h

```diff
@@ -407,17 +408,25 @@ template<class Shape> class ExternalFieldHarmonic : public ExternalFieldMono<Sha
      * acceptance criteria, since it's the energy difference that matters for the latter.
      */
     Scalar
-    calcE(const unsigned int& index, const vec3<Scalar>& position, const quat<Scalar>& orientation)
+    calcE(const unsigned int& index, const vec3<Scalar>& position, const quat<Scalar>& orientation,
+            bool include_translational=true, bool include_rotational=true)
```

Comment:
I prefer not to introduce boolean parameters that get must get passed down the chain. Instead implement `getEnergyTranslational` and `getEnergyRotational` as separate methods.

Reply:
We resolved this in real-time.